### PR TITLE
Add panel Inherited cancer to the clinical masterlist

### PR DESCRIPTION
**Pull request by beatrizsavinhas**
### Added

- Panel `Inherited cancer` to clinical master list

### How to prepare for test

- [x] Ssh to hasta
- [x] Use stage: `us`
- [x] Paxa the environment: `paxa`
- [x] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b Add-Inherited-cancer-to-masterlist -a
    ```

### How to test

- [x] Check if panel is fetched when running a case containing a panel from the masterlist

### Expected test outcome

- [x] Check that `Inherited cancer` appears in the output when running case `justhusky`
- [x] Copy/paste the output.

## Review

- [x] Tests executed by BSV
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Deploy this branch
- [ ] Inform to customer that the change has been deployed

### CODE REVIEW

**Comment by beatrizsavinhas** [comment]
## Testing

```shell
20d [hiseq.clinical@hasta:~] [S_base] 130 $ cg workflow mip-dna start justhusky --dry-run
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Case justhusky exists in Status DB
Starting full MIP analysis workflow for case justhusky
Flow cell check is not applicable - the case is either down sampled or external.
Found file earlycasualcaiman/2019-05-08/HHT5NDSXX_HapMap_S26_L001_R1_001.fastq.gz
Found file earlycasualcaiman/2019-05-08/HHT5NDSXX_HapMap_S26_L001_R2_001.fastq.gz
Found file slowlycivilbuck/2019-05-08/HHT5NDSXX_HapMap_S27_L001_R1_001.fastq.gz
Found file slowlycivilbuck/2019-05-08/HHT5NDSXX_HapMap_S27_L001_R2_001.fastq.gz
Found file hugelymodelbat/2019-05-08/HHT5NDSXX_HapMap_S28_L001_R1_001.fastq.gz
Found file hugelymodelbat/2019-05-08/HHT5NDSXX_HapMap_S28_L001_R2_001.fastq.gz
Case justhusky exists in Status DB
Case justhusky exists in Status DB
Running command /home/proj/stage/bin/miniconda3/envs/S_scout/bin/scout --config /home/proj/stage/servers/config/hasta.scilifelab.se/scout-stage.yaml export panel --bed BRAIN Cardiology CILM CH CTD DIAB ENDO EP HEARING HYDRO IBMFS IEM IF mcarta MHT MIT MOVE mtDNA NBS-M NEURODEG NMD OMIM-AUTO PANELAPP-GREEN PEDHEP PID PIDCAD RETINA SKD SOVM STROKE AID Inherited cancer --build 37
```

`Inherited cancer` loaded!

**Review by northwestwitch — APPROVED** [review]
Looks good! 💯

File: cg/constants/gene_panel.py [hunk 1]
```diff
@@ -41,6 +41,7 @@ class GenePanelMasterList(StrEnum):
     SOVM: str = "SOVM"
     STROKE: str = "STROKE"
     AID: str = "AID"
+    INHERITED_CANCER: str = "Inherited cancer"
```
Comment (seallard) [inline cg/constants/gene_panel.py:44]: This one seems to deviate from the naming pattern of the ones above, is that intended?

**Review by seallard — APPROVED** [review]
:+1:

File: cg/constants/gene_panel.py (hunk 1 above)
Comment (beatrizsavinhas) [reply cg/constants/gene_panel.py:44]: I also thought it was strange and that's why I asked @northwestwitch about it, but this ID seems to have been chosen by the customers as such so it's intended.
This is the panel created by the customer, for reference: https://scout.scilifelab.se/panels/61ee70b121c519723d3f3049

**Comment by beatrizsavinhas** [comment]
```shell
Log deploy... done.
cg, version 54.3.0
```

Deployed successfully.

**Pull request merged**